R/request.R

```diff
@@ -157,6 +157,9 @@ request_perform <- function(req, handle, refresh = TRUE) {
   }
 
   url_scheme <- parse_url(resp$url)$scheme
+  if (is.null(url_scheme))
```

Comment:
Can you please use `{}` as in https://style.tidyverse.org/syntax.html#indenting?